.github/workflows/vcpkg_ci_mac.yml

```diff
@@ -78,10 +78,14 @@ jobs:
           echo "Selecting XCode Version ${{ matrix.os.xcode }}"
           sudo xcode-select -s /Applications/Xcode_${{ matrix.os.xcode }}.app/Contents/Developer
 
+      # Crosscompiling LLVM 16 requires a newer clang than available in XCode 14.2
       - name: Install latest LLVM
+        if: contains(matrix.llvm, 'llvm-16') && matrix.target_arch == 'arm64'
         run: |
           brew install llvm
           echo "/usr/local/opt/llvm/bin" >> "$GITHUB_PATH"
+          echo "CC=/usr/local/opt/llvm/bin/clang" >> "$GITHUB_ENV"
+          echo "CXX=/usr/local/opt/llvm/bin/clang++" >> "$GITHUB_ENV"
```

Comment:
This needs to be added because we no longer set `CC` and `CXX` in `build_dependencies.sh`